src/tests/DisplayCard.test.tsx

```diff
@@ -29,4 +29,30 @@ test('edit card', () => {
 	const text = screen.getByRole('paragraph') as HTMLParagraphElement
 
 	expect(text.textContent).toBe('something')
+
+	const flashCard = screen.getByText('something')
+
+	expect(flashCard).toBeInTheDocument()
+	fireEvent.click(editIcon)
+
+	const deleteIcon = screen.getByAltText('trash icon')
+	fireEvent.click(deleteIcon)
+
+	expect(flashCard).not.toBeInTheDocument()
+})
+test('should display flashCards correctly', () => {
+	const flashCards = [
+		{ front: 'front1', back: 'back1', _id: '1', key: '1' },
+		{ front: 'front2', back: 'back2', _id: '2', key: '2' },
```

Comment:
**issue:** This is incorrect and this is an unit test, not integration test. You should test your component that holds cards, which is probably context provider. Here you just test one component and pass props to it.